doc/source/whatsnew/v1.1.0.rst

```diff
@@ -99,7 +99,7 @@ Other enhancements
   ``df.to_csv(path, compression={'method': 'gzip', 'compresslevel': 1}``
   (:issue:`33196`)
 - :meth:`~pandas.core.groupby.GroupBy.transform` has gained ``engine`` and ``engine_kwargs`` arguments that supports executing functions with ``Numba`` (:issue:`32854`)
--
+- :meth: `melt` has gained an ``index`` argument that prevents the method from dropping the index.
```

Comment:
add the issue number

the ref to melt has an extra space (after the :)

Reply:
Done